spec/functions/is_array_spec.rb

```diff
@@ -21,4 +16,17 @@
   it { is_expected.to run.with_params('one').and_return(false) }
   it { is_expected.to run.with_params(1).and_return(false) }
   it { is_expected.to run.with_params({}).and_return(false) }
+  context 'Checking for deprecation warning' do
```

Comment:
should this be at the bottom ? it being at the top alerts people that this function is about to be deprecated, even in the tests